service/awsconfig/v7/resource/loadbalancer/current.go

```diff
@@ -44,10 +45,17 @@ func (r *Resource) clusterLoadBalancers(customObject v1alpha1.AWSConfig) (*LoadB
 		allLBNames = append(allLBNames, lb.LoadBalancerName)
 	}
 
-	if len(allLBNames) > 0 {
-		// We get tags for all load balancers which needs a separate API call.
+	// Get loadbalancer tags in batches due to API restriction.
+	for i := 0; i < len(allLBNames); i += loadBalancerTagChunkSize {
+		endPos := i + loadBalancerTagChunkSize
+
+		if endPos > len(allLBNames) {
+			endPos = len(allLBNames)
+		}
+
+		lbNames := allLBNames[i:endPos]
 		tagsInput := &elb.DescribeTagsInput{
-			LoadBalancerNames: allLBNames,
+			LoadBalancerNames: lbNames,
```

Comment:
Here I had to reintroduce the fix because I messed up the copy PR.

I'll do a separate PR to improve the test coverage.

Reply:
That is not so cool. Is there something we can do to prevent this in the future?

Reply:
Definitely, it could have easily got missed. 

I think if the copy and import scripts did a git pull that would help. This was user error. There is also the chance that someone else merges and you're not aware.

Reply:
Can you check the wiki and add something?

Reply:
Yes will do.

Reply:
Yes script could do better job at least checking invariants.

Reply:
@kopiczko @xh3b4sd I updated the scripts to get the latest master.

https://github.com/giantswarm/giantswarm/wiki/Releases:-Creating-New-VersionBundle/_compare/b675d67e8a781f782e73d1578f51a6410aebc616...6cbcc095df61ba74c659b1a512e81a9423cc79b7

Reply:
:+1: still, the script lacks some checks. Let's iterate on that.